terminal.go

```diff
@@ -430,30 +456,49 @@ func (g *LightningTerminal) Run() error {
 	case <-readyChan:
 
 	case err := <-g.errQueue.ChanOut():
+		// TODO(positiveblue): set error status for lnd.
```

Comment:
should we maybe not scatter all the TODOs for the status stuff? think it only makes sense if there is actually a status server